server/src/utils/mixerConnections/SSLMixerConnection.ts

```diff
@@ -57,6 +57,119 @@ export class SSLMixerConnection {
         return str.substring(1)
     }
 
+    handleReceivedFaderLevelCommand = (buffer: any) => {

```

Comment:
`buffer` is a very nondescript variable name. What data does this buffer contain? Since your method is called `handleReceivedFaderLevelCommand` I guess that it is a buffer for the fader level. But instead of me having to guess the parameter could just be called `faderLevelBuffer`.

Also, since the name of the method was mentioned. Method names should give a clear description of what the method does. `handleReceivedFaderLevelCommand` does not give an indication of what the method does. It only gives an indication of when the method is called. But the method has no idea of where/when/how many times it is called, so its name should not be refering that.

Lets say you stumble upon this piece of code:
```
<button onClick="() => handleOkButtonClicked()">Ok</button>
```
What will happen when the Ok button is clicked? 
We don't know because `handleOkButtonClicked` refers to when the method is called instead of what the method does. So now we have to go look at the implementation of the method to know what it does.
But if the code instead looked like this:
```
<button onClick="() => acceptCookies()">Ok</button>
```
Here we know that cookies will be accepted when we click the Ok button and we know this without ever looking at the implementation of the method.